src/app/src/main/kotlin/org/vaccineimpact/api/app/app_start/route_config/TouchstoneRouteConfig.kt

```diff
@@ -23,6 +24,15 @@ object TouchstoneRouteConfig : RouteConfig
                     .json()
                     .secure(responsibilityPermissions),
 
+            Endpoint("$baseUrl:touchstone-version-id/responsibilities/comments/", controller, "getResponsibilitiesWithComments")
+                    .json()
+                    .secure(responsibilityReviewPermissions),
+
+            Endpoint("$baseUrl:touchstone-version-id/responsibilities/comments/", controller, "addResponsibilityComment")
```

Comment:
I'm open-minded but I think the case for `addResponsibilityComment` being here is that:
- comments are being attached in the context of a particular touchstone
- keeping the endpoints together feels more maintainable
- they probably both need the same permissions, which are different to those currently applied in `ResponsibilityRouteConfig`

Regarding the route itself, I agree with the comment text should be in the post body as JSON for consistency with other endpoints. For the URL we have the following options:
1. `:touchstone-version-id/responsibilities/comments/` with body of `{comment, group_id, scenario_id}`
2. `:touchstone-version-id/responsibilities/comments/:group-id/:scenario-id/ with body of `{comment}`
3. `:touchstone-version-id/responsibilities/:group-id/:scenario-id/comments/ with body of `{comment}`

There is an argument for each... I would favour option 1 if starting from scratch, but the other two are more consistent with other endpoints. Option 2 would make more sense for a GET, as it would imply returning a subset of comments. Option 3 is more consistent with coverage upload URL structure so I'm inclined to that one. Thoughts welcome!

Reply:
Fair point on the route_config location. 

Yeah, 3 is preferable out of that list I'd say. To be consistent with [getRespnsibility](https://github.com/vimc/montagu-api/blob/master/src/app/src/main/kotlin/org/vaccineimpact/api/app/app_start/route_config/ResponsibilityRouteConfig.kt#L27) it would probably be `/modelling-groups/:group-id/responsibilities/:touchstone-version-id/:scenario-id/comments` - but then that would perhaps make less sense in the context of keeping the route in `TouchstoneRouteConfig`